cmd/convox/start.go

```diff
@@ -55,6 +57,29 @@ func cmdStart(c *cli.Context) error {
 		return stdcli.ExitError(errors.New("could not connect to docker daemon, is it installed and running?"))
 	}
 
+	if os.Getenv("PROVIDER") != "test" {
+		dockerVersionTest, err := docker.NewClientFromEnv()
+		if err != nil {
+			return stdcli.ExitError(err)
+		}
+
+		minDockerVersion, err := docker.NewAPIVersion("1.9")
+		e, err := dockerVersionTest.Version()
+		if err != nil {
+			return stdcli.ExitError(err)
+		}
+
+		currentVersionParts := strings.Split(e.Get("Version"), ".")
```

Comment:
we can not without using a different library or writing our own.
My version is coming out as "1.12.0-rc2" which throws an error something like "0-rc2 can not be converted to an integer"